packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -553,13 +556,12 @@ class _RetainingObjectDescription extends StatelessWidget {
         ).buildTextSpan(context),
       );
     } else {
-      entries.addAll([
-        const TextSpan(text: 'of '),
```

Comment:
Why was this removed?

Reply:
It creates two "of"s in a row, because these lines `entries` always ends with "of" after these lines https://github.com/flutter/devtools/pull/4846/files#diff-8483d57bb25f0c712d2792fc77a3f949a5f9be04fbb779e1e0e966f3a9fd3a64R523-R528